itl/command-plugins.conf

```diff
@@ -549,6 +549,175 @@ object CheckCommand "http" {
+			value = "$curl_no-body$"
+			description = "Don't wait for document body: stop reading after headers. (Note that this still does an HTTP GET or POST, not a HEAD.)"
+		}
+		"-M" = {
+			value = "$curl_max-age$"
+			description = "Warn if document is more than SECONDS old. the number can also be of the form '10m' for minutes, '10h' for hours, or '10d' for days."
+		}
+		"-T" = {
+			value = "$curl_content-type$"
+			description = "specify Content-Type header media type when POSTing"
+		}
+		"-l" = {
+			value = "$curl_linespan$"
+			description = "Allow regex to span newlines (must precede -r or -R)"
+		}
+		"-r" = {
+			value = "$curl_ereg$"
+			description = "Search page for regex STRING"
+		}
+		"-R" = {
+			value = "$curl_eregi$"
+			description = "Search page for case-insensitive regex STRING"
+		}
+		"--invert-regex" = {
+			set_if = "$curl_invert_regex$"
+			description = "When using regex, return CRITICAL if found, OK if not"
+		}
+		"-a" = {
+			value = "$curl_authorization$"
+			description = "Username:password on sites with basic authentication"
+		}
+		"-b" = {
+			value = "$curl_proxy-authorization$"
+			description = "Username:password on proxy-servers with basic authentication"
+		}
+		"-A" = {
+			value = "$curl_user-agent$"
+			description = "String to be sent in http header as 'User Agent'"
+		}
+		"-k" = {
+			value = "$curl_header$"
+			repeat_key = true
+			description = "Any other tags to be sent in http header. Use multiple times for additional headers"
+		}
+		"-E" = {
+			set_if = "$curl_extended-perfdata$"
+			description = "Print additional performance data"
+		}
+		"-B" = {
+			set_if = "$curl_show_body$"
+			description = "Print body content below status line"
+		}
+		"-f" = {
+			value = "$curl_onredirect$"
+			description = "Options: <ok|warning|critical|follow|sticky|stickyport|curl> How to handle redirected pages."
+		}
+		"--max-redirs" = {
+			value = "$curl_max-redirs$"
+			description = "Maximal number of redirects (default: 15)"
+		}
+		"-m" = {
+			value = "$curl_pagesize$"
+			description = "Minimum page size required (bytes) : Maximum page size required (bytes)"
+		}
+		"--http-version" = {
+			value = "$curl_http-version$"
+			description = "Connect via specific HTTP protocol. 1.0 = HTTP/1.0, 1.1 = HTTP/1.1, 2.0 = HTTP/2 (HTTP/2 will fail without -S)"
+		}
+		"--enable-automatic-decompression" = {
+			set_if = "$curl_enable-automatic-decompression$"
+			description = "Enable automatic decompression of body (CURLOPT_ACCEPT_ENCODING)."
+		}
+		"-w" = {
+			value = "$curl_warning$"
+			description = "Response time to result in warning status (seconds)"
+		}
+		"-c" = {
+			value = "$curl_critical$"
+			description = "Response time to result in critical status (seconds)"
+		}
+		"-t" = {
+			value = "$curl_timeout$"
+			description = "Seconds before connection times out (default: 10)"
+		}
+		"-L" = {
+			set_if = "$curl_link$"
+			description = "Wrap output in HTML link (obsoleted by urlize)"
+		}
```

Comment:
This should be located between `-B` and `-f`.

Reply:
Tbh, I quite don't get this option at all.
https://github.com/monitoring-plugins/monitoring-plugins/commit/f4a49a8b34ab3826d4e11ebc31e3709c0536176f

Reply:
Maybe just omit this option with a comment at its position? I really don't see how this can be useful for a CheckCommand executed by Icinga.

Reply:
Honestly just put it there for completeness. It is like a bad idea in any case (although IcingaWeb2 will happily passthrough any HTML in the output).
If you say, you would remove it, I will do that, no strong opinions from my side.